{{cookiecutter.project_name}}/CHANGELOG

```diff
@@ -0,0 +1,8 @@
+=======
```

Comment:
I think we should name this file `CHANGELOG.rst`

Reply:
✔️  Commit 330afbf renames `CHANGELOG` to `CHANGELOG.rst`